toolkit/components/uniffi-bindgen-gecko-js/src/lib.rs

```diff
@@ -64,11 +63,13 @@ impl GeckoJsBindingGenerator {
     }
 
     fn calc_filename(&self, ci: &ComponentInterface) -> String {
+        let basename = format!("{}Scaffolding", ci.namespace().to_camel_case());
```

Comment:
Hmm does this mean the generated jsm file will be called "GeometryScaffolding.jsm"?

Reply:
Oops, I'll fix that.  In practice this doesn't mean much because gecko expects us to output the source code to stdout rather than to a file.